### PR TITLE
Feat#79867

### PR DESCRIPTION
**Pull request by a-morenovalls**
Support to Settings section on the app:

- Custom sequence instead of Standard.
- Vibration support.
- Keep screen on.
- About Scrum Poker section.
- About section updated.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,63 +1,140 @@
...
+    AsyncStorage.getItem('deck').then((deck: string) => {
+      const items = JSON.parse(deck);
+      if (items && items.length) {
+        this.setState({ deck: items });
+      }
+    });
+  }
+
   _onPressCard = () => {
+    AsyncStorage.getItem('vibrate').then((v: string) => {
```
Comment (alvaromb) [inline App.js]: Can you switch from promise to async?

File: App.js [hunk 2]
```diff
@@ -1,63 +1,140 @@
...
 
-class HomeScreen extends React.PureComponent<*, State> {
+class App extends React.PureComponent<*, State> {
   state = {
     hidden: false,
+    deck: [],
   };
 
+  componentDidMount() {
+    AsyncStorage.getItem('deck').then((deck: string) => {
```
Comment (alvaromb) [inline App.js]: Change from promise to async.

File: App.js [hunk 3]
```diff
@@ -1,63 +1,140 @@
...
-            </>
-          )}
-        />
-      </SafeView>
+      <Provider
+        value={{
+          settings: {
+            deck: this.getDeck(),
+          },
+          addItem: (item: string) => {
```
Comment (alvaromb) [inline App.js]: Move functions outside and change from promise to async.

File: App.js [hunk 4]
```diff
@@ -1,63 +1,140 @@
...
+            // Set newItems with state and new item incoming
+            const newItems = [...this.state.deck, item];
+            AsyncStorage.setItem('deck', JSON.stringify(newItems)).then(() => {
+              this.setState({
+                deck: newItems,
+              });
+            });
+          },
+          removeItem: (item: string) => {
```
Comment (alvaromb) [inline App.js]: Move functions outside and change from promise to async.

File: App.js [hunk 5]
```diff
@@ -1,63 +1,140 @@
...
+            const newItems = this.state.deck.filter((v: string) => v !== item);
+            AsyncStorage.setItem('deck', JSON.stringify(newItems)).then(() => {
+              this.setState({
+                deck: newItems.length === 0 ? Deck.Standard : newItems,
+              });
+            });
+          },
+          eraseAll: () => {
```
Comment (alvaromb) [inline App.js]: Move functions outside and change from promise to async.

File: src/Home/Components/About.js [hunk 6]
```diff
@@ -29,73 +34,86 @@ class About extends React.PureComponent<{}> {
...
-    paddingTop: 20,
-    paddingHorizontal: 20,
-    paddingBottom: 20,
   },
   contentContainer: {
     alignItems: 'center',
-  },
-  logo: {
-    padding: 20,
+    paddingBottom: 20,
```
Comment (alvaromb) [inline src/Home/Components/About.js]:
```suggestion
    padding: 20,
```

File: src/Home/Components/Sequence.js [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+    />
+  );
+
+  render() {
+    const { text } = this.state;
+    return (
+      <SafeView>
+        <NavigationBar onPress={this.props.onClose} />
+        <TextInput
+          ref={(ref: any) => {
```
Comment (alvaromb) [inline src/Home/Components/Sequence.js]: Try to use new `createRef` API.

File: src/Home/Components/Settings.js [hunk 8]
```diff
@@ -0,0 +1,244 @@
...
+
+class Settings extends React.PureComponent<Props, State> {
+  state = {
+    screen: 'none',
+    isKeepScreenOn: false,
+    vibrate: false,
+  };
+
+  componentDidMount() {
+    AsyncStorage.multiGet(['screenState', 'vibrate']).then(
```
Comment (alvaromb) [inline src/Home/Components/Settings.js]: async over promise.

**Review by alvaromb — CHANGES_REQUESTED** [review]

**Comment by a-morenovalls** [comment]
Merged into `develop`, releasing a new version to Stores.

**Pull request merged**